src/hotspot/cpu/x86/macroAssembler_x86.hpp

```diff
@@ -291,8 +289,8 @@ class MacroAssembler: public Assembler {
                Register arg_1, Register arg_2, Register arg_3,
                bool check_exceptions = true);
 
-  void get_vm_result  (Register oop_result, Register thread);
-  void get_vm_result_2(Register metadata_result, Register thread);
+  void get_vm_result  (Register oop_result);
+  void get_vm_result_2(Register metadata_result);
```

Comment:
NIT: Can you please try to find a better suffix than "_2"?

Reply:
It is a cross-platform mess. I think we want to rename it in one shot, in a separate PR? Want to take it? This `_2` is basically for metadata, while "`_1`" is for oops.